src/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,52 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-top: rem(24px);
+  margin-bottom: rem(32px);
+
+  animation: slide-in 0.2s ease-in;
+
+  border-radius: rem(12px);
+
+  background-color: rgba(37, 37, 37, 0.6);
+
+  color: #fff;
+  backdrop-filter: blur(1.698rem);
+
+  ul {
+    padding-left: 4.68rem;
+    padding-top: rem(44px);
+    position: relative;
+    opacity: 0.6;
+    font-family: $font-family-sans-serif-normal;
```

Comment:
As I said above, you don't need to set the `font-family` for each element, you just need to set the general font in `html` or `body` tag, then just need to specify the `font-weight` if it's not the normal one.

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/6f7d8911e5efec5fc7ac24ea69f8d8a49e9dda7d#diff-d4c7c98435f46121f3c0148f218bde3512209dfa147e630bf07e76e24e4c2789